src/components/nodes/miscellanious/Preview.vue

```diff
@@ -38,7 +41,8 @@ export default {
   },
   data: () => ({
     value: null,
-    type: undefined
+    type: undefined,
+    lightbox: false
```

Comment:
Perhaps the lightbox logic should be separated into a new Vue component so that the Preview node is less cluttered. This would theoretically also decrease memory consumption, as it is filtered out by `v-if` in cases where the type is not `image`.

Reply:
I agree for structural purposes, however the `q-dialog` will already efficiently save memory whenever it's not shown.